### ISSUE TITLE
Installing on Server 2016 Core

### ISSUE THREAD

**Issue by tresf** [msys2/msys2#18]
Currently, the Msys2 installer requires OpenGL which prevents it from being installed on Server 2016 Core.

```
The program cannot start because OPENGL32.dll is missing from your computer.
Try reinstalling the program to fix this problem
```

I managed to extract the archive with 7z to `C:\msys2` and then ran `C:\msys2\msys2_shell.cmd` to setup the environment.
* Is this the right way to install programatically?  I was considering writing a [scoop bucket](https://github.com/lukesampson/scoop-extras) for msys2 and this would drastically simplify the environment setup. :)
* Are there plans to create an installer that does not depend on OpenGL so that it can be launched either headless or at least on Server Core?

**Comment by mati865** [comment]
> Is this the right way to install programatically?

Yes, you have to extract `msys2-base-*.tar.xz` archive from https://sourceforge.net/projects/msys2/files/Base/x86_64/
Unfortunately it requires you to have tar and xz tools, they come with mentioned 7-zip for an example.

**Comment by martell** [comment]
In general this seems like something that should be reported to upstream `qt-installer-framework` as a bug.

Also I think you can run the installer quietly without loading the gui.
I believe this is done on the drone-ci stuff the guys here work on.

**Comment by tresf** [comment]
> this seems like something that should be reported to the upstream qt bug tracker.
They should want the framework to support Server 2016 Core without having to do headless.

Is [headless](https://bugreports.qt.io/browse/QTIFW-166) an all-or-nothing flag?  Headless should be just fine/preferred if it can be flagged "on" with parameter(s).  This would also help inclusion in the package management and CI environments by avoiding volatile 7zip CLI steps.

These installer framework instructions mention a [`-no-opengl` flag by name](https://github.com/qtproject/installer-framework/blob/master/INSTALL#L31) as a recommendation for windows.  Is this related?

> Does adding `-platform minimal` work for you?

I tried the following command to no avail.  Am I doing it right?

```batch
msys2-x86_64-20161025.exe -platform minimal
```

Research on this command suggests [it can automate the installation with the addition of a JavaScript file](http://blog.qt.io/blog/2015/04/07/qt-installer-framework-2-0-released/#comment-1195532), which could help, but the same OpenGL error occurs.
